products/BellHybrid/apps/application-bell-relaxation/model/RelaxationSongsRepository.hpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+#include <apps-common/models/SongsRepository.hpp>
```

Comment:
Please separate `#pragma`s from `#include`s and namespace declarations by an empty line, that way it's easier to read